config/samples/image_signature/README.md

```diff
@@ -0,0 +1,144 @@
+```
+
+You are asked to enter information that will be incorporated in your certificate
+request.
+
+
+```bash
+$ openssl x509 -req -days 14 -in cert_request.csr -signkey private_key.pem -out new_cert.crt
+Signature ok
+```
+
+- Upload the certificate in `Barbican` that will be used to store the secret:
+
+```bash
+$ openstack secret store --name test --algorithm RSA --secret-type certificate \
+  --payload-content-type "application/octet-stream" \
+  --payload-content-encoding base64 \
+  --payload "$(base64 new_cert.crt)"
+
++---------------+-----------------------------------------------------------------------+
+| Field         | Value                                                                 |
++---------------+-----------------------------------------------------------------------+
+| Secret href   | http://127.0.0.1:9311/v1/secrets/cd7cc675-e573-419c-8fff-33a72734a243 |
++---------------+-----------------------------------------------------------------------+
+```
+
+Get the `cert_uuid` from the secret generated in the previous step:
+
+```bash
+$ cert_uuid=cd7cc675-e573-419c-8fff-33a72734a243
+```
+
+- Get an image and create the signature.
+
+```bash
+$ echo This is a dodgy image > myimage
+
+$ openssl dgst -sha512 -sign private_key.pem -sigopt rsa_padding_mode:pss -out myimage.signature myimage
+
+$ base64 -w 0 myimage.signature > myimage.signature.b64
+
+$ image_signature=$(cat myimage.signature.b64)
+
+$ cat myimage.signature.b64
+  KFYrU1XIFXC6RyYO9AhYB4VN1vnm+ImZyEuYLHXzgebs7sJGIdoLfb4ELnfIHI5Ijm/H/zUiY0BQGMbrzzoFUQQkOacu7oU24CxDLP3XrjI0nE9d1qmCH6m81/eVeEhyp/NCKi7NSYaadp4jCMI1gLyNZMbQCBKWeT784/4jPgs=
+```
+
+
+- Create image with valid signature properties
+
+```bash
+$ glance --os-auth-url {{ KeystoneAuthUrl }} \
+         --os-project-name {{ project }} --os-username {{ user }} --os-password {{ pwd }} \
+         --os-user-domain-name default --os-project-domain-name default \
+         image-create --name mySignedImage --container-format bare --disk-format qcow2 \
+         --property img_signature="$image_signature" --property img_signature_certificate_uuid="$cert_uuid" \
+         --property img_signature_hash_method='SHA-512' --property img_signature_key_type='RSA-PSS' < myimage
+```
+
+### Note:
+
+Replace all the variables of the `image-create` command with the data provided
+by the `clouds.yaml` file. The `clouds.yaml` file can be either retrieved by:
+
+```bash
+oc get cm openstack-config -o json | jq -r '.data["clouds.yaml"]'
+```
+
+or, if the `openstackclient` `Pod` is used to perform such commands, it can be
+found in the `$HOME/.config/openstack/` path.
+
+The password passed to the `--os-password` option can be retrieved by querying the `OpenStack` secret.
+```bash
+oc get secrets osp-secret -o jsonpath="{ .data.AdminPassword }" | base64 -d
+```
+
+- osp-secret: the secret used to store the passwords associated to the deployed
+  ctlplane
+
+
+## Known Limitations
+
```

Comment:
I think in known issues we should also mention failure if provide sha256 at line 95 while creating digest.

Reply:
Done